### PR TITLE
Add getter to days attribute.

### PR DESCRIPTION
**Pull request by BerRRG**
Hello, i add a new getter to return the list of days.

### CODE REVIEW

File: src/Carbon/Carbon.php [hunk 1]
```diff
@@ -946,6 +946,16 @@ public function setTimezone($value)
         return parent::setTimezone(static::safeCreateDateTimeZone($value));
     }
 
+    /**
+     * Get days
```
Comment (lucasmichot) [inline src/Carbon/Carbon.php]: Maybe `Get the days of the week.`

Comment (BerRRG) [inline src/Carbon/Carbon.php]: Fixed, thanks.

**Comment by lucasmichot** [comment]
Just rebooting Travis

**Comment by BerRRG** [comment]
@lucasmichot Travis is failing is anything i can fix ?

**Comment by lucasmichot** [comment]
@BerRRG, not worries
This is not related to the code of your PR.

I will fix that in another PR

**Comment by lucasmichot** [comment]
@BerRRG , feel free to rebase your PR against master

**Comment by BerRRG** [comment]
@lucasmichot, Its done, Thanks

**Comment by Webysther** [comment]
@lucasmichot Is it possible to release a patch (maybe 1.22.2?) to remove the composer pointing to `dev-master`?

**Pull request merged**